Src/Setup/BuildInstaller.bat

```diff
@@ -58,6 +58,17 @@ light Temp\Setup64.wixobj -nologo -out Temp\Setup64.msi -ext WixUIExtension -ext
 @if ERRORLEVEL 1 exit /b 1
 
 
+REM ********* Build ARM64 MSI
+echo --- ARM64 MSI
+candle Setup.wxs -nologo -out Temp\SetupARM64.wixobj -ext WixUIExtension -ext WixUtilExtension -dx64=0 -dARM64=1 -dCS_LANG_FOLDER=%CS_LANG_FOLDER% -dCS_LANG_NAME=%CS_LANG_NAME%
```

Comment:
So it seems we cannot do this by default as there is no proper WiX support on `AppVeyor`.
Either just comment it for now. Or add some parameter that could be used when building locally.